src/routes/artistRouter.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const {
+  createArtist,
+  readArtist,
+  getArtistById,
+  updateArtist,
+  deleteArtist,
+} = require("../controllers/artistController");
+
+router.post("/", createArtist);
+
+router.get("/", readArtist);
+
+router.get("/:id", getArtistById);
+
+router.put("/:id", updateArtist);
+
+router.delete("/:id", deleteArtist);
```

Comment:
In this case since theres not that many routes I would say you could put these together without sacrificing readability
```
router.post("/", createArtist);
router.get("/", readArtist);
router.get("/:id", getArtistById);
router.put("/:id", updateArtist);
router.delete("/:id", deleteArtist);
```